### ISSUE TITLE
Discussion about batch_size

### ISSUE THREAD

**Issue by volgachen** [lucidrains/titans-pytorch#36]
As to my understanding, the gradients on multiple chunks are calculated in parallel during training for accelerating. In inference, the weight should be immediately updated after each chunk.

In the repo, I found the weights is updated only after a batch of chunks in inference, the same as in training. I wonder which would be better.
